.github/workflows/e2e_test_spice_cli.yml

```diff
@@ -0,0 +1,245 @@
+name: E2E Test CLI behavior
+on:
+  workflow_dispatch:
+    inputs:
+        rel_version:
+          description: 'A dummy release version used for building the binaries'
```

Comment:
We should have an option to test with the real previous Spice CLI instead of building a fake older version with the latest code.